test/integration/cli/server-process-cli-test.coffee

```diff
@@ -90,6 +90,26 @@ describe 'CLI - Server Process', ->
       it 'should exit with status 0', ->
         assert.equal dreddCommandInfo.exitStatus, 0
 
+    describe 'When it fails to start', ->
```

Comment:
My attempt is in https://github.com/apiaryio/dredd/pull/862. If the section screenshoted  above won't get green, then the test is useless and my PR should get closed.